src/Engine/babylon.engine.ts

```diff
@@ -5396,4 +5393,16 @@
             }
         }
     }
+
+    export namespace Engine {

```

Comment:
We are not using enum as they are generating string and string comparison.
Just add statics in the Engine

Reply:
Wanted to add strong typing rather than just `: number`; but yeah enum does add a lot of trash

Reply:
Yeah I know :(